package/archlinux/PKGBUILD-release

```diff
@@ -25,6 +25,7 @@ build() {
         -DBUILD_GL_TESTS=ON \
         -DWITH_ASSIMPIMPORTER=ON \
         -DWITH_BASISIMPORTER=ON \
+        -DWITH_BASISIMAGECONVERTER=OFF \
```

Comment:
Where Importer is `ON`, converter could be `ON` also :) I guess you ran into conflicts with all those?

Reply:
You mean during rebase? Yes 😅

Reply:
I enabled the converter on archlinux/* PKGBUILD files except emscripten and mingw.